opengrok-tools/requirements.txt

```diff
@@ -6,7 +6,7 @@ jsonschema==2.6.0
 JsonSir==0.0.2
 Python-EasyConfig==0.1.7
 PyYAML==3.13
-requests==2.19.1
+requests~=2.20.0
```

Comment:
if you need to force some version for the package, it must be in setup.py

this file is only a snapshot of the exact dev environment where this package was developed and not a file with requirements to install.

Reply:
Thing is the Github security vulnerability scanner interprets the `requirements.txt` file when emitting warnings.

Reply:
Ah. But then you shoud make it available also for users in setup.py, that's how you enforce a specific version range in the installation phase.

Reply:
okay. It's strange to have this information stored in two places.

Reply:
This requirements.txt is a generated file and the only purpose is to for some other developer to get the exact copy of your dev environment.

The source of truth is the setup.py.